src/how-to-do-links-to-other-pages.wiki

```diff
@@ -1,28 +1,63 @@
 =How to do links to other pages?=
 
-Examples:
+Sometimes, you need to create links to other websites. For Eliom, these links are called «external services». These external services are services which are not registered and generated by your application (for example a link to https://ocsigen.org).
```

Comment:
I would much prefer the tutorial to start with standard internal services, these come up more frequently. And then you can say that it's easy to use the same mechanism for external links, with examples like the ones you have already.